solidity/dashboard/src/components/coverage-pools/ExchangeRate.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react"
+import { Keep } from "../../contracts"
+import TokenAmount from "../TokenAmount"
+
+const BaseExchangeRate = ({
+  covToken,
+  amount,
+  htmlTag = "div",
+  className = "",
+}) => {
+  const Tag = htmlTag
+  return (
+    <Tag className={className}>
+      {`1 ${covToken.symbol}`} = ~
+      <TokenAmount
+        amount={amount}
+        decimalsToDisplay={3}
+        wrapperClassName="flex-inline"
+        amountClassName=""
+        symbolClassName=""
+      />
+    </Tag>
+  )
+}
+
+export const CoveragePoolV1ExchangeRate = ({
+  covToken,
+  collateralToken,
+  covTotalSupply,
+  totalValueLocked,
+  ...restProps
+}) => (
+  <BaseExchangeRate
+    {...restProps}
+    covToken={covToken}
+    amount={Keep.coveragePoolV1.estimatedBalanceFor(
+      collateralToken.fromTokenUnit(1).toString(),
```

Comment:
Do we actually need `collateralToken` prop here? We are using `KEEP.coveragePoolV1.estimateBalanceFor` one line above.

So either change above line to `collateralToken.coveragePoolV1.estimateBalanceFor` or remove the `collateralToken` prop from `CoveragePoolV1ExchangeRate` and keep it as `KEEP.fromTokenUnit(1).toString()`

Reply:
The main idea is to be independent of KEEP token because coverage pools can work with any ERC20 token. I've updated a bit the `estimateBalanceFor` method and left a TODO. Please see c49944d.